src/app/components/ui/Footer/index.tsx

```
@@ -0,0 +1,123 @@
+  return (
+    <footer className="border-t bg-background">
+      <div className="container py-8 md:py-12">
+        <div className="grid gap-8 sm:grid-cols-2 md:grid-cols-4">
+          <div>
+            <h3 className="mb-4 text-lg font-semibold">About</h3>
+            <ul className="space-y-3 text-sm">
+              <li>
+                <a
+                  href="/about"
+                  className="text-muted-foreground hover:text-foreground"
+                >
```

ESLint

[error] Do not use an `<a>` element to navigate to `/about/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)